docs/user-guide/stopwords.rst

```diff
@@ -0,0 +1,179 @@
+Stop Words in Text Processing
+=============================
+
+Stop words are common words that are often filtered out in natural language processing (NLP) tasks because they typically don't carry significant meaning. Examples in English include "the," "is," "at," "which," and "on." These words appear so frequently in language that they can distort text processing tasks if not properly managed. NVIDIA NeMo Curator provides built-in stop word lists for several languages to support text analysis and extraction processes.
+
+Key characteristics of stop words:
+
+* They appear with high frequency in text
+* They typically serve grammatical rather than semantic functions
+* They're language-specific (each language has its own set of stop words)
+* Removing them can improve efficiency in many NLP tasks
+
+Why Stop Words Matter in NeMo Curator
+------------------------------------
+
+In NeMo Curator, stop words play several important roles:
+
+* **Text Extraction**: The text extraction process (especially for Common Crawl data) uses stop word density as a key metric to identify meaningful content
+* **Language Detection**: Stop words help in language detection and processing
+* **Efficient Processing**: Filtering stop words reduces the amount of data that needs to be processed
+* **Boilerplate Removal**: Stop word density helps differentiate between main content and boilerplate in web pages
+
+Available Stop Word Lists
+------------------------
+
+NeMo Curator includes built-in stop word lists for the following languages:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 50 30
+
+   * - Language
+     - File Name
+     - Number of Stop Words
+   * - Thai
+     - ``thai_stopwords.py`` and ``th_stopwords.py``
+     - ~120
+   * - Japanese
+     - ``ja_stopwords.py``
+     - ~140
+   * - Chinese
+     - ``zh_stopwords.py``
+     - ~800
+
+These stop word lists are implemented as Python frozen sets for efficient lookup and immutability.
+
+Thai Stop Words
+~~~~~~~~~~~~~~
+
+Thai stop words are available in two files: ``thai_stopwords.py`` and ``th_stopwords.py``. Both contain around 120 common Thai words like "กล่าว" (to say), "การ" (the), and "ของ" (of).
```

Comment:
There is only `th_stopwords.py` (which was originally called `thai_stopwords.py` but the original name does not need to be mentioned in documentation).